HomeTask16/src/test/java/hillel/qaauto/SeleniumDevBase.java

```diff
@@ -0,0 +1,69 @@
+package hillel.qaauto;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+import java.time.Duration;
+import java.util.List;
+
+public class SeleniumDevBase {
+
+    WebDriver driver;
+    WebDriverWait wait;
+    final String SELENIUM_DEV_URL = "https://www.selenium.dev/selenium/web/dynamic.html";
+    final String EACH_BOX_COLOR = "red";
+    String addBoxButtonXpath = "//input[@id='adder']";
+    String revealNewInputButtonXpath = "//input[@id='reveal']";
+    String inputFieldXpath = "//input[@id='revealed']";
+    String allBoxesXpath = "//div[@class='redbox']";
```

Comment:
You can use simplified locators for elements search, e.g.
   String addBoxButtonId = "adder"; 
then:
 driver.findElement(By.id(addBoxButtonId));